src/runtime/vdom/vdom-render.ts

```diff
@@ -1027,7 +1064,7 @@ render() {
             // has a different next sibling or parent relocated
 
             if (nodeToRelocate !== insertBeforeNode) {
-              if (!nodeToRelocate['s-hn'] && nodeToRelocate['s-ol']) {
+              if (!BUILD.experimentalSlotFixes && !nodeToRelocate['s-hn'] && nodeToRelocate['s-ol']) {
```

Comment:
Idea is that this whole block will go away once the slot fix flag is defaulted. Setting `s-hn` for elements from `index.html` actually causes some weird behavior when putting nodes back since it won't relocate a node to the same host node (which we actually want in some scenarios). 

I didn't see any negative consequences from removing this while playing around with things, and all our automated tests still pass so feel like we're okay to do this.